MotorStatus front_left
MotorStatus front_right
MotorStatus rear_left
MotorStatus rear_right
